ClearKeep/Scenes/ChatGroup/Views/CommonUI/ListUser.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ListUser.swift
+//  ClearKeep
+//
+//  Created by đông on 30/03/2022.
+//
+
+import SwiftUI
+import Common
+import CommonUI
+
+private enum Constant {
+	static let sizeImage = 64.0
+}
+
+struct ListUser: View {
+	@State private(set) var isSelectedUser: Bool = false
+
+    var body: some View {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/3249a8332ccef754f5404bc6377103b48f3d2d68/ClearKeep/Scenes/ChatGroup/Views/CommonUI/ListUser.swift#L19"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>ListUser.swift:19</code></td>
    </tr>
  </tbody>
</table>